### PR TITLE
feat: profiles moved to layouts

### PR DESCRIPTION
**Pull request by milorue**

### CODE REVIEW

File: pages/profiles/[publicKey]/nfts.tsx [hunk 1]
```diff
@@ -365,7 +364,7 @@ enum ListingFilters {
 export const INFINITE_SCROLL_AMOUNT_INCREMENT = 25;
 export const INITIAL_FETCH = 25;
 
-const ProfileNFTs: NextPage<WalletDependantPageProps> = (props) => {
+const ProfileNFTs = (props: WalletDependantPageProps) => {
```
Comment (austinmilt) [inline pages/profiles/[publicKey]/nfts.tsx:367]: `function`

File: pages/profiles/[publicKey]/nfts.tsx [hunk 2]
```diff
@@ -490,60 +489,63 @@ const ProfileNFTs: NextPage<WalletDependantPageProps> = (props) => {
...
+            onFocus={() => setSearchFocused(true)}
+            onBlur={() => setSearchFocused(false)}
+          />
+          <GridSelector />
+        </div>
+      </div>
+      <NFTGrid
+        ctaVariant={`collected`}
+        hasMore={hasMore && filteredNfts.length > INITIAL_FETCH - 1}
+        onLoadMore={async (inView) => {
```
Comment (austinmilt) [inline pages/profiles/[publicKey]/nfts.tsx:531]: would be awesome to extract this logic outside of the markup

File: src/common/components/elements/ProfileContainer.tsx [hunk 3]
```diff
@@ -86,12 +86,12 @@ export const ProfileContainer: FC = ({ children }) => {
 const ProfileDisplayName: FC = () => {
   const { publicKey, twitterHandle } = useProfileData();
 
-  const [copied, setCopeied] = useState(false);
+  const [copied, setCopied] = useState(false);
```
Comment (austinmilt) [inline src/common/components/elements/ProfileContainer.tsx:89]: nice

File: src/common/components/layouts/ProfileLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+const Banner = styled.div`
+  width: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  ${mq('lg')} {
+    background-size: 100%;
+  }
+`;
+
+// export const getServerSideProps: GetServerSideProps<WalletDependantPageProps> = async (context) => {
```
Comment (austinmilt) [inline src/common/components/layouts/ProfileLayout.tsx:214]: delete?

**Review by austinmilt — COMMENTED** [review]
Mostly just pointing out opportunities to do minor refactoring based on guidelines discussions

File: pages/profiles/[publicKey]/activity.tsx [hunk 5]
```diff
@@ -1,31 +1,25 @@
...
-        pfp: props.profilePicture,
-      }}
-      description="View activity for this, or any other pubkey, in the Holaplex ecosystem."
-    />
-    <ProfileContainer>
-      <ActivityContent />
-    </ProfileContainer>
-  </ProfileDataProvider>
-);
+const ActivityPage = ({ publicKey, ...props }: WalletDependantPageProps) => <ActivityContent />;
```
[the remark this answers is not in the document]
Comment (milorue) [reply pages/profiles/[publicKey]/activity.tsx:13]: cool resolved as per slack convo

File: pages/profiles/[publicKey]/created.tsx [hunk 6]
```diff
@@ -140,90 +137,89 @@ const CreatedNFTs: NextPage<WalletDependantPageProps> = (props) => {
...
-                />
-              }
-              placeholder="Search"
-              onFocus={() => setSearchFocused(true)}
-              onBlur={() => setSearchFocused(false)}
-            />
-            <GridSelector />
-          </div>
+    <>
+      <div className="sticky top-0 z-10 flex flex-col items-center gap-6 bg-gray-900 py-4 lg:flex-row lg:justify-between lg:gap-4">
```
[the remark this answers is not in the document]
Comment (milorue) [reply pages/profiles/[publicKey]/created.tsx:141]: yeah i agree just to make it a bit more readable

File: pages/profiles/[publicKey]/created.tsx [hunk 7]
```diff
@@ -140,90 +137,89 @@ const CreatedNFTs: NextPage<WalletDependantPageProps> = (props) => {
...
+        <div className={`flex w-full justify-start gap-4 lg:items-center`}>
+          <ListingFilter title={`All`} filterToCheck={ListingFilters.ALL} count={totalCount} />
```
[the remark this answers is not in the document]
Comment (milorue) [reply pages/profiles/[publicKey]/created.tsx:143]: I would keep these out of a form for now but a refactor could be warranted to that effect

File: pages/profiles/[publicKey]/nfts.tsx (hunk 2 above)
Comment (milorue) [reply pages/profiles/[publicKey]/nfts.tsx:531]: yesir

File: src/common/components/layouts/ProfileLayout.tsx (hunk 4 above)
Comment (milorue) [reply src/common/components/layouts/ProfileLayout.tsx:214]: yesir

**Pull request merged**